source/documentation/deploying_services/index.md

```diff
@@ -650,13 +650,235 @@ Note that data restore will not be available in the event of an RDS outage that
 
 For more details about how the RDS backup system works, see [Amazon's DB Instance Backups documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.BackingUpAndRestoringAmazonRDSInstances.html) [external page].
 
-## MongoDB
+## Redis
 
-MongoDB is an open source cross-platform document-oriented database program. It uses JSON-like documents with schemas, and is often used for content management such as articles on [GOV.UK](https://www.gov.uk/). This is an early version of the service that is available on request so that we can get feedback, and we will make you aware of any constraints in its use at that time.
+Redis is an open source in-memory data store that can be used as a database cache or message broker. This is a public beta version of the service that is available on request so that we can get feedback. We will make you aware of any constraints in its use at time of request.
```

Comment:
We need to mention that this is "private beta" as the first line in this section, and we should also mention _how_ to contact us to request access (probably via email)